README.md

```diff
@@ -37,92 +38,11 @@ from the sensor, but return the values read last. To read a new sample, make
 sure to call `readSample()`
 
 ### Sample code
```

Comment:
Sample implementations, as there is no code anymore?

Reply:
the code is just one link down, see the rendered version here: https://github.com/Sensirion/arduino-sht/blob/0c4b3319655d9165446dfd1566cabef09878e79f/README.md

Reply:
Ah, like this? https://github.com/Sensirion/arduino-sht/blob/c9ef8b3a9197879c8f63b9edf3ed364e96d67a3f/README.md